Realm/RLMMongoCollection.mm

```diff
@@ -93,31 +95,41 @@ - (void)close {
 
 @implementation RLMMongoCollection
 
-- (instancetype)initWithApp:(RLMApp *)app
-                serviceName:(NSString *)serviceName
-               databaseName:(NSString *)databaseName
-             collectionName:(NSString *)collectionName {
+static realm::bson::BsonDocument toBsonDocument(id<RLMBSON> bson) {
```

Comment:
Should this be prefixed by `RLM`?

Reply:
It's static, and only exported functions should be prefixed.